WebRoot/messages/ZmMsg_fr_CA.properties

```diff
@@ -2143,6 +2143,18 @@ oAuthAppDevice = Appareil
 oAuthAppApprovedDate = Approuv\u00e9
 removeOAuthAppAuthorization = Supprimer l\u2019autorisation de l\u2019application
 oAuthAppAuthorizationRemoveConfirm = Cette action aura pour effet de supprimer l\u2019autorisation accord\u00e9e \u00e0 \u00ab\u00a0{0}\u00a0\u00bb, sur votre \u00ab\u00a0{1}\u00a0\u00bb.
+mobileConFigurationDownloadLabel =	T\u00e9l\u00e9charger
+mobileConfiguration = Configuration mobile
+mobileConfigurationAll = Email, calendrier et t\u00e2ches, contacts
+mobileConfigurationCaldav =	Calendrier et t\u00e2ches
+mobileConfigurationCarddav = Contacts
+mobileConfigurationDav = Calendrier et t\u00e2ches, contacts
+mobileConfigurationDownload = Vous pouvez t\u00e9l\u00e9charger un profil mobileconfig pour configurer automatiquement les appareils et les applications qui le prennent en charge.
+mobileConfigurationDownloadList = Configuration pour:
+mobileConfigurationImap = Email
+mobileConfigurationInstruction = IMAP, CalDAV et CardDAV sont des protocoles ouverts permettant de synchroniser les ressources de votre compte avec des appareils mobiles ou des applications de bureau.
+mobileConfigurationOptions = <div><p>IMAP pour e-mail</p><p>CalDAV pour le calendrier et les t\u00e2ches</p><p>& bull; & ensp; CardDAV pour les contacts</p></div>
```

Comment:
make mobileConfigurationOptions propperly `<div><p>&bull;&ensp;` is missing

Reply:
Added.

Reply:
mobileConfigurationOptions  = `<div><p>IMAP pour e-mail</p><p>CalDAV pour le calendrier et les t\u00e2ches</p><p>& bull; & ensp; CardDAV pour les contacts</p></div>` 

this should be like 
mobileConfigurationOptions  = `<div><p>&bull;&ensp;IMAP pour e-mail</p><p>&bull;&ensp;CalDAV pour le calendrier et les t\u00e2ches</p><p>& bull; & ensp; CardDAV pour les contacts</p></div>`

Reply:
Done.